docs/tutorial-osparc.ipynb

```diff
@@ -0,0 +1,437 @@
+    "try:\n",
+    "  from tqdm import tqdm\n",
+    "except ImportError:\n",
+    "  ! pip install tqdm\n",
+    "from pprint import pprint\n",
+    "import os\n",
+    "from tqdm import tqdm\n",
+    "from sparc.client import SparcClient\n",
+    "from sparc.client.services.o2sparc import (\n",
+    "  O2SparcService,\n",
+    "  O2SparcSolver\n",
+    ")\n",
+    "from time import sleep\n",
+    "from pathlib import Path\n",
+    "from tempfile import TemporaryDirectory"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Login to the o²S²PARC Python Client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set-up credentials for o²S²PARC in osparc.io as explained [here](https://docs.osparc.io/#/docs/platform_introduction/user_setup/security_details?id=generating-o%c2%b2s%c2%b2parc-tokens). Then use the \"Key\" as \"Username\" and \"Secret\" as \"Password\". Another option is to export them as environment variables."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "osparc host, e.g. https://api.osparc.io ········\n",
+      "API Key: ········\n",
+      "API Secret: ········\n"
+     ]
+    }
+   ],
+   "source": [
+    "os.environ[\"O2SPARC_HOST\"] = getpass.getpass(\"osparc host, e.g. https://api.osparc.io\")\n",
+    "os.environ[\"O2SPARC_USERNAME\"] = getpass.getpass(\"API Key:\")\n",
+    "os.environ[\"O2SPARC_PASSWORD\"] = getpass.getpass(\"API Secret:\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "assert \"O2SPARC_HOST\" in os.environ, \"O2SPARC_HOST must be exposed as an environment variable\"\n",
+    "assert \"O2SPARC_USERNAME\" in os.environ, \"O2SPARC_USERNAME must be exposed as an environment variable\"\n",
+    "assert \"O2SPARC_PASSWORD\" in os.environ, \"O2SPARC_PASSWORD must be exposed as an environment variable\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Access to the computational services provided by the `o2sparc` submodule, is provided via `O2SparcService`, an instance of which can be creates as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = SparcClient(connect=False, config_file='config.ini')\n",
+    "o2sparc: O2SparcService = client.o2sparc"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As a sanity check you can now contact the Osparc server to check that you are logged in as the correct user:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "iavarone@itis.swiss\n"
```

Comment:
Personal details visible here, is this the intention?

Reply:
Thanks for pointing this out. As mentioned above I have cleared all outputs.